src/pages/Home.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+    Box,
+    Body,
+    Tiles,
+} from "@buildo/bento-design-system";
+import RestaurantPreview from "../components/RestaurantPreview";
+import rest_detail from "../mock-data/rest_detail.json";
+
+function Home() {
+    const name = rest_detail.name;
+    const image_url = rest_detail.image_url;
+    const rating = rest_detail.rating;
+    const address = rest_detail.location.address1;
+
+    const mockCards = [...Array(16)].map(() => {
+        return <RestaurantPreview
+            name={name}
+            image_url={image_url}
+            rating={rating}
+            address={address}
+        />
+    })
```

Comment:
This is minor, but when building an array of the same component, you should give each one a unique `key` attribute ([ref](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)): right now this only serves to silence those noisy warnings you get in console, but in the long run this is important if your list can change (e.g. by sorting or by inserting/deleting elements).